### PR TITLE
add speakers corner talk instructions

### LINKED ISSUE #179: Speakers' corner speaker checklist
Once the talk is approved, the bot should respond to the speaker with a standard checklist of actions that explains how everything works.

I'll start writing some notes here, please chip in.

### PR DESCRIPTION
**Pull request by akhmerov**
Closes #179.

I'd really appreciate a second look at the instructions here.

### CODE REVIEW

**Comment by akhmerov** [comment]
Following @vfatemi suggestion I have disabled local recording for the speakers' corner user. This simplifies the instructions.

File: templates/talk_registered.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+We've added your talk to the schedule :tada:!
+
+Below please find a checklist for your presentation.  
+Please read it carefully and confirm that everything is clear.  
+Good luck with your talk!
+
+## Before your meeting
+
+- We will send you an email containing the meeting key and your join URL in a moment.
+  **Confirm that you received the meeting key and URL by responding to this email.**
```
Comment (akhmerov) [inline templates/talk_registered.md:10]: This isn't an email, rather the bot response in the issue.

```suggestion
  **Confirm that you received the meeting key and URL by responding in this issue.**
```

**Comment by akhmerov** [comment]
@vfatemi anything else standing out?

**Comment by vfatemi** [comment]
@akhmerov Additional suggestions made.

**Comment by akhmerov** [comment]
All looks good, merge?

**Pull request merged**